src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -329,7 +329,7 @@ public function reset()
      */
     public function getServiceIds()
     {
-        return array_map('strval', array_unique(array_merge(['service_container'], array_keys($this->fileMap), array_keys($this->methodMap), array_keys($this->aliases), array_keys($this->services))));
+        return array_map('\strval', array_unique(array_merge(['service_container'], array_keys($this->fileMap), array_keys($this->methodMap), array_keys($this->aliases), array_keys($this->services))));
```

Comment:
wrong change.
This made me wonder: how did you spot the changed lines in the PR? Did you use eg php-cs-fixer?

Reply:
I reverted this change, as it was really not expected here - but I'd not call it wrong.

As reference the used php-cs-rule which also got applied to phpunit: https://github.com/sebastianbergmann/phpunit/commit/1a9dd1a392c6670a241e9e737cc385e50bbc6f7b) - but basically the same changes got suggested by PHP Inspections EA ([this rule](https://github.com/kalessil/phpinspectionsea/blob/b83ce4aa06c3a650dd93a1fec7d051a8dde45a72/src/main/java/com/kalessil/phpStorm/phpInspectionsEA/inspectors/apiUsage/UnqualifiedReferenceInspector.java))

-> [as example here a similar `array_map('\trim', explode(',', $color));`](https://github.com/sebastianbergmann/phpunit/blob/e15dfe6ebddacd7ed093b008332f775a4cc077f3/src/Util/Color.php#L84)

But only kept files which had a reduction of opcodes (after opcache optimizer phase) using a simple script which compared the version before and after.